diff --git a/live-neighbour-root.tmp b/live-neighbour-root.tmp
index 7947ad392d..ff9a3658e7 100644
--- a/live-neighbour-root.tmp
+++ b/live-neighbour-root.tmp
@@ -37,3 +37,24 @@ edit 36
 edit 37
 edit 38
 edit 39
+edit 40
+edit 41
+edit 42
+edit 43
+edit 44
+edit 45
+edit 46
+edit 47
+edit 48
+edit 49
+edit 50
+edit 51
+edit 52
+edit 53
+edit 54
+edit 55
+edit 56
+edit 57
+edit 58
+edit 59
+edit 60
